.github/workflows/hitl.yml

```diff
@@ -1,9 +1,9 @@
 name: HITL
 
 on:
-  push:
-    branches: [ hitl ]
   workflow_dispatch:
+  pull_request:
+    branches: [ master ]
```

Comment:
Then we can also add a cron schedule here. The only downside is appears to be the deploy review requests piling up. Let's see if we can handle that.

Also I think the best approach would be to have a `repository-dispatch` action that waits for the dispatch to complete for the token to be valid long enough. I don't know whether the ones available are stable and high quality enough to get us there. Alternatively let's create a PAT on `quartiq/hitl` that has write access. Might be less hassle.

Reply:
1. Let's do the cron job in the `hitl` repository so that we don't need to approve the jobs. It should be relatively easy there.
2. There's a lot of complexity involved in making a `repository-dispatch` wait for the scheduled workflow to complete. How do you even know the `repository-dispatch` generates a job? What if it generates multiple? I think the simplest solution here is to just create a persistent PAT. They're easy to revoke if you don't want to continue using it in the future.

Reply:
Agreed on both.

But the problem with the cron job on `hitl` is that it doesn't have the dispatch object data and might be tricky to write/duplicate.

I think you'd just wait for the check that `hitl` would cross-post here (not wait for the job). Like `https://github.com/fountainhead/action-wait-for-check` or similar. But yes. Let's do PAT first and get the check link cross-posted first.